v0/destinations/facebook_offline_conversions/config.js

```diff
@@ -0,0 +1,103 @@
+const { getMappingConfig } = require("../../util");
+
+const ENDPOINT = "https://graph.facebook.com/v15.0/OFFLINE_EVENT_SET_ID/events";
+
+const CONFIG_CATEGORIES = {
+  OFFLINE_EVENTS: {
+    type: "track",
+    name: "FbOfflineConversionsTrackConfig"
+  }
+};
+
```

Comment:
Add this ref: https://developers.facebook.com/docs/marketing-api/offline-conversions/#upload-events here